conda_forge_tick/env_management.py

```diff
@@ -0,0 +1,31 @@
+import os
+from contextlib import contextmanager
+
+
+class SensitiveEnv():
+    SENSITIVE_KEYS = ["USERNAME", "PASSWORD", "GITHUB_TOKEN", "GH_TOKEN"]
+
+    def __init__(self):
+        self.clasified_info = {}
+        self.classify()
+
+    def classify(self):
+        self.clasified_info.update({k: os.environ.pop(k, None) for k in self.SENSITIVE_KEYS})
+
+    def declassify(self):
+        self.clasified_info.update(
+            {k: os.environ.pop(k, None) for k in list(self.clasified_info)},
+        )
```

Comment:
Actually looking at this I am confused. doc strings would help me.

Reply:
declassify is not the opposite of classify. It is doing something else. Maybe `reclassify` or `classify_known_keys`?